### ISSUE TITLE
About the use of validation set

### ISSUE THREAD

**Issue by JoeYing1019** [downdric/MSD#4]
Hello, according to your code, it seems that the model is saved based on the best results from the test set, rather than the results from the validation set?

**Comment by downdric** [comment]
We sincerely appreciate your interest in our work. During the training procedure, please utilize "text_data/valid.txt" for accessing the validation set at line 210 of the main.py script. Our codebase is equipped to monitor metrics on the validation set throughout the training phase.
